tests/common/dualtor/mux_simulator_control.py

```diff
@@ -322,3 +322,16 @@ def _drop_helper(interface_name):
 
     yield _drop_helper
     set_output(tmp_list[0], [UPPER_TOR, LOWER_TOR])
+
+
+def toggle_mux_simulator_to_tor(host, mux_server_url, tbinfo):
```

Comment:
Can we re-use this fixture here? https://github.com/Azure/sonic-mgmt/blob/3615c82a1a7c6e2b2585bf8e93ce3fcaba2df0ef/tests/common/dualtor/mux_simulator_control.py#L262

Reply:
The reason for adding this function is to toggle mux state during testing. It's hard to do that with a fixture.

Reply:
The fixture I linked returns a function, I believe we started using this pattern so that we could control when to execute specific actions during a test case.

Reply:
Indeed. But it would be easier to toggle mux cable to a given tor with this helper function. We can also do that with fixture ```toggle_all_simulator_port```, but it's not as convenient since we have to check if the ```rand_selected_tor``` is ```UPPER_TOR``` or ```LOWER_TOR```.

Reply:
I would still prefer to move this logic into `set_mux_state` and re-use the above fixture, to avoid duplicate code here.

Reply:
Moving into ```set_mux_state``` is a good idea. Updated. Thanks